Examples/Python/tests/helpers/__init__.py

```diff
@@ -33,19 +33,19 @@
     except ImportError:
         dd4hepEnabled = False
 
-hepmc3Enabled = "HepMC3_DIR" in os.environ
-if hepmc3Enabled:
-    try:
-        import acts.examples.hepmc3
-    except ImportError:
-        hepmc3Enabled = False
+try:
+    import acts.examples.hepmc3
 
-edm4hepEnabled = "EDM4HEP_DIR" in os.environ
-if edm4hepEnabled:
-    try:
-        import acts.examples.edm4hep
-    except ImportError:
-        edm4hepEnabled = False
+    hepmc3Enabled = True
+except ImportError:
+    hepmc3Enabled = False
```

Comment:
Would it make sense to also change the above `geant4Enabled` (l.8-13) and `dd4hepEnabled` (l.29-34) to this cleaner version? (as also already done with `rootEnabled`)

Reply:
The underlying issue is that `edm4hep` and `hepmc3` weren't importing additional modules in the helper python files. Geant4, DD4hep and ROOT are not functional without the runtime environment variables set, so this makes sense to have differently in my opinion.